src/shared/vscode/secrets.ts

```diff
@@ -0,0 +1,58 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// Sourced from https://github.com/microsoft/vscode/blob/238c19cc7727511d495d67d2ce1e0ec0638de2ce/src/vscode-dts/vscode.d.ts
+// This API was added in 1.53. Not supported in Cloud9.
+declare module 'vscode' {
```

Comment:
This API should (hopefully) be supported by C9 whenever they move to 1.53 so I'm just adding the type definition now.